### PR TITLE
allow fine-tuning of timing for lorawan-device

### PR DESCRIPTION
**Pull request by lthiery**

### CODE REVIEW

**Comment by plaes** [comment]
Could you try building with the `#![allow(missing_docs)]` removed on the first line?

**Comment by lthiery** [comment]
> Could you try building with the #![allow(missing_docs)] removed on the first line?

I added that when I moved the module from lorawan-device to lora-phy. There are some undocumented structs...

File: lora-phy/src/lorawan_radio.rs [hunk 1]
```diff
@@ -29,22 +34,36 @@ where
...
+}
+
 /// Provide the timing values for boards supported by the external lora-phy crate
 impl<RK, DLY, const P: u8, const G: i8> Timings for LorawanRadio<RK, DLY, P, G>
 where
     RK: RadioKind,
     DLY: DelayNs,
 {
-    // TODO: Use a struct with Default or Option fallback?
     fn get_rx_window_offset_ms(&self) -> i32 {
```
Comment (lucasgranberg) [inline lora-phy/src/lorawan_radio.rs:62]: Could you add a comment on what this offset and duration actually does?

Comment (lthiery) [reply lora-phy/src/lorawan_radio.rs:62]: The comment exists on the trait definition already: https://github.com/lora-rs/lora-rs/blob/main/lorawan-device/src/lib.rs#L61-L70

Comment (lucasgranberg) [reply lora-phy/src/lorawan_radio.rs:62]: Whoops, didn't see that

**Review by lucasgranberg — APPROVED** [review]

**Pull request merged**